sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/finetuning/azure_openai_finetuning_job.py

```diff
@@ -0,0 +1,228 @@
+from azure.ai.ml._restclient.v2024_01_01_preview.models import (
+    ModelProvider as RestModelProvider,
+    AzureOpenAiFineTuning as RestAzureOpenAIFineTuning,
+    FineTuningJob as RestFineTuningJob,
+    JobBase as RestJobBase,
+)
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+from azure.ai.ml.entities._job._input_output_helpers import from_rest_data_outputs, to_rest_data_outputs
+from typing import Any, Dict
+from azure.ai.ml.entities._job.finetuning.finetuning_vertical import FineTuningVertical
+from azure.ai.ml.entities._job.finetuning.azure_openai_hyperparameters import AzureOpenAIHyperparameters
+from azure.ai.ml.entities._util import load_from_dict
+from azure.ai.ml.exceptions import ErrorCategory, ErrorTarget, ValidationException
+
+
+class AzureOpenAIFineTuningJob(FineTuningVertical):
```

Comment:
all the classes inside entities which are Public in pythonic way, should have  @eperiment annotation. https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2?path=/docs/SDK_preview_features.md&_a=preview&anchor=%40experimental-usage-examples

Reply:
Updated.